.github/workflows/ci.yml

```diff
@@ -64,13 +67,14 @@ jobs:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
 
-      - name: Build x86 image
+      - name: Build multi-platform image
         uses: docker/build-push-action@v4
         with:
           context: .
           tags: "ehco1996/ehco:latest"
           push: true
           file: build/Dockerfile
+          platforms: linux/amd64,linux/arm64
 
       # TODO fix the arm build error  gcc: error: unrecognized command-line option '-marm'; did you mean '-mabm'?
       # - name: Build arm image
```

Comment:
how about  remove those todos in this pr?

Reply:
👌